xray/utils/analyzermanager.go

```diff
@@ -0,0 +1,130 @@
+package utils
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	analyzerManagerFilePath  = "analayzerManager/analyzerManager"
```

Comment:
You should use `filepath.Join` to build this path, instead of using a plain slash, so that this works properly on Windows.